magma/simulator/mdb.py

```diff
@@ -30,6 +30,9 @@ def __init__(self, string, vars, scope):
     def display(self, value):
         print("\t{}: {} = {}".format(self.idx, self.string, value))
 
+def print_err(str):
+    print(str, file=sys.stderr)
+
```

Comment:
`magma` has a `logging` module that we're trying to move all the logging code to use (so that we can easily improve the logging infrastructure without having to change as much code). I think this can just use `logging.error`.

Reply:
Actually since `mdb`is a frontend that should own stdout, I think it's okay to have its own logging infrastructure if you think it will have different behavior.

Reply:
What about all the normal prints?

Reply:
I'll leave it for now then, we can always revisit. I'm not sure if there will be different behavior for the frontend in the future or not as far as error logging goes.